src/OpenTelemetry.Exporter.OpenTelemetryProtocol/README.md

```diff
@@ -18,9 +18,9 @@ dotnet add package OpenTelemetry.Exporter.OpenTelemetryProtocol
 
 ## Configuration
 
-You can configure the `OtlpExporter` through `OtlpExporterOptions`
-properties and environment variables. The `OtlpExporterOptions`
-setters take precedence over the environment variables.
+You can configure the `OtlpExporter` through `Options` types properties
```

Comment:
Would it still make sense to use the actual type name instead of the generic `Options`? Just thinking if this won't make things harder for people looking at these instructions. How do I know the type I can use? Need to dig into the code maybe? 🤔

Reply:
Sounds good, was more just some food for though!